### ISSUE TITLE
文字识别转为inference模型后，出现('', nan)

### ISSUE THREAD

**Issue by fdshan** [PaddlePaddle/PaddleOCR#3746]
未转为inference时，识别效果很好。
`[2021/08/19 07:35:22] root INFO: load pretrained model from ['./output/rec_chinese_lite_v2.0/best_accuracy']
[2021/08/19 07:35:23] root INFO: infer_img: ./doc/antique_book/ZHG030108500002_crop_0.jpg
[2021/08/19 07:35:24] root INFO: 	 result: ('同', 0.95341927)
[2021/08/19 07:35:24] root INFO: infer_img: ./doc/antique_book/ZHG030108500002_crop_1.jpg
[2021/08/19 07:35:24] root INFO: 	 result: ('地', 0.99022186)
[2021/08/19 07:35:24] root INFO: infer_img: ./doc/antique_book/ZHG030108500002_crop_2.jpg
[2021/08/19 07:35:24] root INFO: 	 result: ('坤', 0.8744374)
[2021/08/19 07:35:24] root INFO: success!`

转化成inference后，出现('', nan):
`[2021/08/19 07:46:11] root INFO: Predicts of ./doc/antique_book/ZHG030108500002_crop_0.jpg:('', nan)
[2021/08/19 07:46:11] root INFO: Predicts of ./doc/antique_book/ZHG030108500002_crop_1.jpg:('', nan)
[2021/08/19 07:46:11] root INFO: Predicts of ./doc/antique_book/ZHG030108500002_crop_2.jpg:('', nan)
[2021/08/19 07:46:11] root INFO: Total predict time for 3 images, cost: 0.889`

**Comment by littletomatodonkey** [comment]
导出日志发来看下，感觉像是导出的时候没有正确加载

**Comment by fdshan** [comment]
`!python3 tools/export_model.py -c configs/rec/ch_ppocr_v2.0/rec_chinese_lite_train_v2.0.yml -o Global.pretrained_model=./output/rec_chinese_lite_v2.0/best_accuracy  Global.save_inference_dir=./inference/rec_chinese_lite`

`{'Global': {'debug': False, 'use_gpu': True, 'epoch_num': 500, 'log_smooth_window': 20, 'print_batch_step': 10, 'save_model_dir': './output/rec_chinese_lite_v2.0', 'save_epoch_step': 3, 'eval_batch_step': 1000, 'cal_metric_during_train': True, 'pretrained_model': './output/rec_chinese_lite_v2.0/best_accuracy', 'checkpoints': None, 'save_inference_dir': './inference/rec_chinese_lite', 'use_visualdl': False, 'infer_img': 'doc/imgs_words/ch/word_1.jpg', 'character_dict_path': './ppocr/utils/word_dict_ch.txt', 'character_type': 'ch', 'max_text_length': 14, 'infer_mode': False, 'use_space_char': True, 'save_res_path': './output/rec/predicts_chinese_lite_v2.0.txt'}, 'Optimizer': {'name': 'Adam', 'beta1': 0.9, 'beta2': 0.999, 'lr': {'name': 'Cosine', 'learning_rate': 0.001}, 'regularizer': {'name': 'L2', 'factor': 1e-05}}, 'Architecture': {'model_type': 'rec', 'algorithm': 'CRNN', 'Transform': None, 'Backbone': {'name': 'MobileNetV3', 'scale': 0.5, 'model_name': 'small', 'small_stride': [1, 2, 2, 2]}, 'Neck': {'name': 'SequenceEncoder', 'encoder_type': 'rnn', 'hidden_size': 48}, 'Head': {'name': 'CTCHead', 'fc_decay': 1e-05}}, 'Loss': {'name': 'CTCLoss'}, 'PostProcess': {'name': 'CTCLabelDecode'}, 'Metric': {'name': 'RecMetric', 'main_indicator': 'acc'}, 'Train': {'dataset': {'name': 'SimpleDataSet', 'data_dir': './train_data/', 'label_file_list': ['./train_data/rec/rec_gt_train.txt'], 'transforms': [{'DecodeImage': {'img_mode': 'BGR', 'channel_first': False}}, {'RecAug': None}, {'CTCLabelEncode': None}, {'RecResizeImg': {'image_shape': [3, 32, 320]}}, {'KeepKeys': {'keep_keys': ['image', 'label', 'length']}}]}, 'loader': {'shuffle': True, 'batch_size_per_card': 256, 'drop_last': True, 'num_workers': 8}}, 'Eval': {'dataset': {'name': 'SimpleDataSet', 'data_dir': './train_data/', 'label_file_list': ['./train_data/rec/rec_gt_test.txt'], 'transforms': [{'DecodeImage': {'img_mode': 'BGR', 'channel_first': False}}, {'CTCLabelEncode': None}, {'RecResizeImg': {'image_shape': [3, 32, 320]}}, {'KeepKeys': {'keep_keys': ['image', 'label', 'length']}}]}, 'loader': {'shuffle': False, 'drop_last': False, 'batch_size_per_card': 256, 'num_workers': 8}}}
W0819 09:41:36.121382 29046 device_context.cc:362] Please NOTE: device: 0, GPU Compute Capability: 6.0, Driver API Version: 11.2, Runtime API Version: 10.2
W0819 09:41:36.125293 29046 device_context.cc:372] device: 0, cuDNN Version: 7.6.
[2021/08/19 09:41:37] root INFO: load pretrained model from ['./output/rec_chinese_lite_v2.0/best_accuracy']
[2021/08/19 09:41:38] root INFO: inference model is saved to ./inference/rec_chinese_lite/inference`

**Comment by fdshan** [comment]
> 导出日志发来看下，感觉像是导出的时候没有正确加载

发在上面啦。

**Comment by littletomatodonkey** [comment]
你好，看着没啥问题，方便把预训练模型发我邮箱吗？我本地测试下 dazhiningsibuqu@163.com
